alwaysontop/main.js

```diff
@@ -64,9 +64,9 @@ function onAlwaysOnTopWindow(
                 titleBarStyle: undefined,
                 frame: false,
                 show: false,
-                webPreferences: {
+                webPreferences: Object.assign({}, options.webPreferences, {
```

Comment:
Why?

Reply:
to keep webPreferences set upstream, not overwrite them

Reply:
I figured that. My question is more like:
How this relates to the PR? Are you trying yo solve an issue with this? What is the issue? What is the need of this change?

If this is not related to the PR, please open another PR with this change and put some description there what are you trying to solve!

Reply:
it was just something I noticed while trying different approaches - I'll revert this change